### PR TITLE
Ond232 149 create custom phonemizer with

### PR DESCRIPTION
**Pull request by andrey999333**
new servicer to make CRUD operation with custom phonemizer

### CODE REVIEW

**Review by rinjac — APPROVED** [review]

File: ondewo/t2s/custom_phonemizer.proto [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+// endpoints of custom phonemizer
+service CustomPhonemizers{
+    rpc GetCustomPhonemizer (PhonemizerId) returns (CustomPhonemizerProto) {
+    };
+    rpc CreateCustomPhonemizer (CreateCustomPhonemizerRequest) returns (PhonemizerId) {
+    };
+    rpc DeleteCustomPhonemizer (PhonemizerId) returns (google.protobuf.Empty) {
+    };
+    rpc UpdateCustomPhonemizer (UpdateCustomPhonemizerRequest) returns (CustomPhonemizerProto) {
```
Comment (AzzerAce) [inline ondewo/t2s/custom_phonemizer.proto:25]: why does `UpdateCustomPhonemizer` return `CustomPhonemizerProto`, while `CreateCustomPhonemizer` returns only `PhonemizerId`?

File: ondewo/t2s/custom_phonemizer.proto [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    string id = 1;
+    repeated Map maps = 2;
+}
+
+message Map{
+    string word = 1;
+    string phoneme_groups = 2;
+}
+
+message ListCustomPhomenizerResponse {
```
Comment (AzzerAce) [inline ondewo/t2s/custom_phonemizer.proto]:
```suggestion
message ListCustomPhonemizerResponse {
```

File: ondewo/t2s/custom_phonemizer.proto [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    repeated CustomPhonemizerProto phonemizers = 1;
+}
+
+message ListCustomPhomenizerRequest{
```
Comment (AzzerAce) [inline ondewo/t2s/custom_phonemizer.proto]:
```suggestion
message ListCustomPhonemizerRequest{
```

**Review by AzzerAce — CHANGES_REQUESTED** [review]
a few grammatical errors, one design question

File: ondewo/t2s/custom_phonemizer.proto (hunk 1 above)
Comment (andrey999333) [reply ondewo/t2s/custom_phonemizer.proto:25]: > why does `UpdateCustomPhonemizer` return `CustomPhonemizerProto`, while `CreateCustomPhonemizer` returns only `PhonemizerId`?

When you create new phenemizer you send it to the server, so the only new thing server does is create new id. Therefore there is no need to send the whole phonemizer back, client has it already, except id, that will be sent. On the other hand, if you update phonemizer, depending on the update mode you get a phonemizer that client does not have yet - you just add some new words for example. So it make sense to send it back.

**Pull request merged**